lib/core/include/irods/base64.h

```diff
@@ -1,14 +1,8 @@
-/*** Copyright (c), The Regents of the University of California            ***
- *** For more information please refer to files in the COPYRIGHT directory ***/
-
-/* base64.h - header file for base64.c
- */
-
-
-
 #ifndef BASE64_H__
 #define BASE64_H__
```

Comment:
I believe @korydraughn mentioned renaming these header guards to `IRODS_BASE64_H`

Reply:
He mentioned it for the HPP one.  I was intending to leave he old deprecated one alone although I did remove the first few lines.

Reply:
I suppose since I have already touched it I might as well rename the header guard.

Reply:
done